plugins/catalog/src/components/AboutCard/AboutContent.tsx

```diff
@@ -99,6 +103,16 @@ export function AboutContent(props: AboutContentProps) {
   });
   const ownedByRelations = getEntityRelations(entity, RELATION_OWNED_BY);
 
+  function getPathForFilter(filter: string, value: string) {
```

Comment:
Best I can tell there is nothing that's actually testing this function, can we make sure to add tests for this, please?

Reply:
I see the existing unit tests only test for text content, so I didn't make any changes to that.

I'll try to add tests for the links.